students/lee_deitesfeld/lesson04/mailroom.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+
+import sys
+from operator import itemgetter
+
+main_prompt = "\n".join(("Please choose from below mailroom options:",
+          "1 - Manage Donors and Donations",
+          "2 - Create a Report",
+          "3 - Send letters to all donors",
+          "4 - Quit",
+          ">>> "))
+
+
+sub_prompt = "\n".join(("Please make a selection from the following options",
+                        "1 - List the Names of Donors",
+                        "2 - Add a New Donation",
+                        "3 - Return to Main Menu"
+                        ">>> "))
+
+dic_donors = {'donor_db':
+                [
+                {"name": "Bob Jones",
+                 "donations": [63.00, 30.00]
+                },
+                {"name": "Sue Smith",
+                 "donations": [50.00]
+                },
+                {"name": "Joe Grimes",
+                 "donations": [102.00, 45.00, 70.00]
+                },
+                {"name": "Andrea Funk",
+                 "donations": [21.00, 21.00]
+                },
+                {"name": "Mickey Mouse",
+                 "donations": [500.00, 250.00, 100.00]
+                }
+                ]}
+
+def menu_selection(prompt, main_dispatch):
+    while True:
+        response = input(prompt)
+        if main_dispatch[response]() == "exit menu":
+            break
+
+def sub_menu():
+    menu_selection(sub_prompt, sub_dispatch)
+
+def quit():
+    print("Quitting this menu")
+    return "exit menu"
```

Comment:
you can you use here `sys.exit()` and then you won't need to check for "exit menu" string.